src/components/MovieTile/MovieTile.tsx

```diff
@@ -26,21 +27,26 @@ const MovieTile: React.FC<MovieTileProps> = ({movieDetails, onClick, handleEdit,
 
     return (
         <div className={styles.movieTile}
-             data-testid="movieTile"
+             data-testid="movie-tile"
              onMouseEnter={() => setHovered(true)}
              onMouseLeave={() => setHovered(false)}>
-            <img
-                 src={imageUrl}
+            <img className={styles.movieTileImage}
+                 src={!poster_path ? placeholderImage : poster_path}
                  alt={title}
-                 onClick={() => onClick(movieDetails)} />
+                 onClick={() => onClick(movieDetails)}
+                 onError={(e) => {
+                     e.currentTarget.src = placeholderImage;
+                     e.currentTarget.onerror = null;
+                 }
+                 } />
```

Comment:
separate handler. You can leave as is, just recommendation, For all places where we can do it.